chain/kusama/defaults.go

```diff
@@ -4,56 +4,53 @@
 package kusama
 
 import (
-	"github.com/ChainSafe/gossamer/internal/log"
+	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/ChainSafe/gossamer/dot/state/pruner"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/genesis"
 	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
 )
 
 var (
-	// GlobalConfig
-
 	// DefaultName Default node name
-	DefaultName = string("Kusama")
+	DefaultName = "Kusama"
```

Comment:
Do we still need to export all these values from this package?

Reply:
No, I've removed it. Thank you!